src/FormField/RegisteredMFAMethodListField.php

```diff
@@ -39,8 +39,16 @@ public function getSchemaData()
     {
         $adminController = AdminRegistrationController::singleton();
         $generator = SchemaGenerator::create();
-        /** @var Member $member */
-        $member = DataObject::get_by_id(Member::class, $this->value);
+
+        if (!$this->value && $this->getForm() && $this->getForm()->getRecord()) {
+            $this->value = $this->getForm()->getRecord()->ID;
+        }
+        if (!$this->value && $this->getForm() && $this->getForm()->getRecord() instanceof Member) {
```

Comment:
This won't actually fetch the member if the previous condition populated `$this->value`.

Reply:
Whoops - I'm rushing, sorry